MTM-45067 added description for DTM to cockpit docs (#1511)
SoftwareAG/c8y-docs: content/cockpit/managing-assets-bundle/asset-hierarchy.md

* **Groups**: Objects which group single devices or other groups. Groups can either be created in the Cockpit application or in the Device management application.

> * **Custom assets**: Objects defined by an asset model and created in the [Digital twin manager](/dtm/dtm-introduction/).

Reviewer: ```suggestion
* **Custom assets**: Objects defined by an asset model and created in the [Digital twin manager](/dtm/dtm-introduction/) application.
```

If you manage trucks within the {{< product-c8y-iot >}} platform, then each truck is represented via its individual tracking device communicating with {{< product-c8y-iot >}}.

For building management, it is most common that a group of sensors inside a building represents the building as a group communicating with the {{< product-c8y-iot >}} platform.

> If more properties and better visualization of business assets is needed, we propose to use the [Digital twin manager](/dtm/dtm-introduction/). A custom asset created via digital twin manager is also shown in the Cockpit application. It uses the chosen icon and additionally shows all the properties in a list next to the sub-assets. Those properties can be edited if the user has the inventory write permission.

Reviewer: ```suggestion
If you need more properties and better visualization of business assets, we recommend you to use the [Digital twin manager](/dtm/dtm-introduction/) application. A custom asset created via the Digital twin manager application is also shown in the Cockpit application. It uses the chosen icon and additionally shows all properties in a list next to the subassets. These properties can be edited if the user has WRITE permission for the permission type "Inventory".
```
